docs/pages/foundations/international_design/number_localization.js

```diff
@@ -0,0 +1,517 @@
+      <Table.Row>
+        <Table.Cell>
+          <Text size="200" align="end">১৭</Text>
+        </Table.Cell>
+        <Table.Cell>
+          <Text size="200" align="end">১২৩.০০US$</Text>
+        </Table.Cell>
+        <Table.Cell>
+          <Text size="200" align="end">২,০৯১.০০US$</Text>
+        </Table.Cell>
+      </Table.Row>
+
+      <Table.Row>
+        <Table.Cell>
+          <Text size="200" align="end">১</Text>
+        </Table.Cell>
+        <Table.Cell>
+          <Text size="200" align="end">৪.৫৬US$</Text>
+        </Table.Cell>
+        <Table.Cell>
+          <Text size="200" align="end">৪.৫৬US$</Text>
+        </Table.Cell>
+      </Table.Row>
+
+    </Table.Body>
+  </Table>
+</Box>
+</Box>
+
+<Box maxWidth={DOCS_COPY_MAX_WIDTH_PX} marginTop={8}>
+<Markdown
+            text={`
+            The same numbers represented in the Western Arabic and Bengali numeral systems  `}
+          />
+</Box>
+
+<Box maxWidth={DOCS_COPY_MAX_WIDTH_PX} marginTop={10}>
+<MainSection.Subsection
+          title="Fonts"
+          description={`Fonts factor into our number localization as well. When we select fonts for our digital products, we may want to make sure that they support all the numeral systems we need. Most operating systems today do a good job of falling back to system fonts when they encounter characters that the currently active font doesn't support. This means that we can often use custom fonts that don't include all the numeral systems we support without numbers turning into gibberish on us. </br>
+          However, it's worth considering international numeral systems when doing UX research for our apps and websites. An international stock exchange app may need to represent all kinds of numeral systems, for example, and our font selection process may need to be sensitive to this. In all cases, we should always test our apps and websites on different platforms to see how our numeral systems are being rendered for our users.`}
+      />
+</Box>
+
+<MainSection.Subsection
+          title="Separators: Breaking up large numbers"
+          description={`When representing large numbers, it often helps with legibility to break up the numbers into digit groups. In many English locales we're used to separating thousands with a comma, and to separate the integer and fraction parts of a number with a point—like **1,234,567.89**. However, it's important to be aware that different locales separate digit groups differently. In French (and even in some English locales), for example, it's common to separate groups of thousands with space and to separate an integer and a fraction with a comma—like **1 234 567,89**. Some locales only separate the integer portion of a number, while others separate both the integer and fractional portions. </br>
+          In case you're interested in best practices on how to format numbers at Pinterest, check out our [Content Standards](https://gestalt.pinterest.systems/foundations/content_standards/voice).`}
```

Comment:
Unless we're intentionally trying to have links open in a new window, we should use relative links for all links within our domain. So this should just use 
```js
[Content Standards](/foundations/content_standards/voice)
```